### DISCUSSION TITLE
TypeScript unable to find declaration file

### DISCUSSION THREAD

**Discussion by jgosmann** [gpbl/react-day-picker#2568] [Support]
Starting with version 9.2 of react-day-picker, the TypeScript compiler complains with:

```
node_modules/react-day-picker/dist/esm/formatters/formatMonthDropdown.d.ts:12:17 - error TS7016: Could not find a declaration file for module 'date-fns/locale/types.js'. '/Users/redacted/project-root/node_modules/date-fns/locale/types.js' implicitly has an 'any' type.

12 locale?: import("date-fns/locale/types.js").Locale): string;
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/react-day-picker/dist/esm/formatters/formatMonthDropdown.d.ts:12
```

## Expected Behavior

No errors from the TypeScript compiler.

## Actual Behavior

Above error from the TypeScript compiler.

**Comment by gpbl** [comment]
@jgosmann please how can I reproduce it? 🙏

**Reply by jgosmann** [reply]
Here is a minimal reproduction: [jgosmann/react-day-picker-type-error-reproduction](https://github.com/jgosmann/react-day-picker-type-error-reproduction)

Why was this moved to a discussion? To me this seems very much like a bug.

**Reply by jgosmann** [reply]
@gpbl friendly ping

**Reply by gpbl** [reply]
@jgosmann thanks for the repo for reproducing it. For some reason the TypeScript compiler was resolving the type that way. I could fix it in https://github.com/gpbl/react-day-picker/pull/2584

> Why was this moved to a discussion? To me this seems very much like a bug.

Recently, the repo is getting issues that are actually support requests. If I am unable to reproduce the bug, I move those issues to the discussion section. Not sure if this is the most effective approach for resolving issues :) 

In your case, I had questions (such as which TypeScript version is used? Is the date-fns package installed? etc.) that you could answer in that repository. Thank you!
